projects/projen.io/js/package.yml

```diff
@@ -0,0 +1,38 @@
+distributable:
+  url: https://github.com/projen/projen/archive/refs/tags/v{{version.raw}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: projen/projen
+  strip: /^v/
+
+dependencies:
+  nodejs.org: ^22
+  classic.yarnpkg.com: ^1
```

Comment:
is this a runtime or a build-time dep?

Reply:
Projen is used to bootstrap & maintain projects, it executes yarn during runtime for JS/TS projects.

Reply:
Generally, a program requires a dependency to execute, most often libraries. A companion is usually another program, which may be optional, that provides functionality.

Companion functionality is newer than dependency functionality, and there are, guaranteed, many violations of this split in the repo.